docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -68,12 +68,14 @@ Property Name                                              Description
 ``kafka.messages-per-split``                               Number of messages that are processed by each Trino split, defaults to 100000
```

Comment:
Oh... wow. I see a lot of my comments addressed here.

Please squash commits then and mark yourself as co-author. This commit on its own does not make much sense in this PR.

Please also address all my comments from previous commit that were not addressed here and ping me please.

 If there are still some comments not addressed

Reply:
Sorry for the confusion. I wasn't aware of this GH feature about co-author. I just merged the 2 commits!